gtsfm/frontend/verifier/degensac.py

```diff
@@ -71,24 +73,33 @@ def verify(
         if match_indices.shape[0] < self._min_matches:
             return self._failure_result
 
-        i2Fi1, mask = pydegensac.findFundamentalMatrix(
+        i2Fi1, inlier_mask = pydegensac.findFundamentalMatrix(
             keypoints_i1.coordinates[match_indices[:, 0]],
             keypoints_i2.coordinates[match_indices[:, 1]],
-            px_th=PIXEL_COORD_RANSAC_THRESH,
-        )
-
-        inlier_idxes = np.where(mask.ravel() == 1)[0]
-
-        i2Ei1_matrix = verification_utils.fundamental_to_essential_matrix(
-            i2Fi1, camera_intrinsics_i1, camera_intrinsics_i2
-        )
-
-        i2Ri1, i2Ui1 = verification_utils.recover_relative_pose_from_essential_matrix(
-            i2Ei1_matrix,
-            keypoints_i1.coordinates[match_indices[inlier_idxes, 0]],
-            keypoints_i2.coordinates[match_indices[inlier_idxes, 1]],
-            camera_intrinsics_i1,
-            camera_intrinsics_i2,
+            px_th=self._estimation_threshold_px,
         )
 
-        return i2Ri1, i2Ui1, match_indices[inlier_idxes]
+        inlier_idxs = np.where(inlier_mask.ravel() == 1)[0]
+
+        v_corr_idxs = match_indices[inlier_idxs]
+        inlier_ratio_est_model = np.mean(inlier_mask)
```

Comment:
I think inlier_ratio is sufficient for the name.

Reply:
This is not a ratio. This should be `inlier_sum`?

Reply:
@ayushbaid thanks for the review. I didn't quite follow here -- I believe this is a ratio -- #ransac inliers / #putative matches.

Reply:
Oh I got it now. Thanks!